server/cluster/cluster.go

```diff
@@ -265,7 +265,13 @@ func (s *Server) Delete(ctx context.Context, q *cluster.ClusterQuery) (*cluster.
 	if err := s.enf.EnforceErr(ctx.Value("claims"), rbacpolicy.ResourceClusters, rbacpolicy.ActionDelete, createRBACObject(c.Project, c.Server)); err != nil {
 		return nil, err
 	}
-	err = s.db.DeleteCluster(ctx, q.Server)
+
+	server := q.Server
+	if server == "" {
```

Comment:
also please move it to private function that will return selector

Reply:
I added to the ClusterSelector as it seems this may be a common function for supporting using cluster name. Happy to make local/private though.